group.yml

```diff
@@ -179,7 +180,7 @@ repos:
         ppc64le: http://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel8/ppc64le/fast-datapath/os/
         s390x: http://rhsm-pulp.corp.redhat.com/content/dist/layered/rhel8/s390x/fast-datapath/os/
         # FDP has yet to be released for aarch64
```

Comment:
Should this comment be removed too?

Reply:
No, not yet, FDP hasn't been released and we still need to use puddles in the meantime.